projects/lights_control/inc/lights_config.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "can.h"
+#include "gpio.h"
+
+#define LIGHTS_CAN_RX_ADDR \
+  { GPIO_PORT_B, 6 }
+#define LIGHTS_CAN_TX_ADDR \
+  { GPIO_PORT_B, 7 }
+
+// declaration of all the configuration variables
+const CANSettings can_settings_front;
```

Comment:
You are likely to get multiply defined variable errors doing this. Consider instead providing accessors to static memory for these variables.